@navikt/aksel/src/codemod/run-codeshift.ts

```diff
@@ -28,15 +28,31 @@ export async function runCodeshift(
     `./transforms/${getMigrationPath(input)}.js`,
   );
 
-  const filepaths = fg.sync([options.glob ?? getDefaultGlob(options?.ext)], {
+  console.info(chalk.greenBright.bold("\nWelcome to Aksel codemods!"));
+  console.info("\nRunning migration:", chalk.green("input"));
+
+  const globList = [options.glob ?? getDefaultGlob(options?.ext)];
+
+  console.info(
+    chalk.gray(
+      `Using glob pattern(s): ${globList.join(", ")}\nWorking directory: ${process.cwd()}\n`,
```

Comment:
`globList` doesn't need to be an array, it seems.

```suggestion
  const globList = options.glob ?? getDefaultGlob(options?.ext);

  console.info(
    chalk.gray(
      `Using glob pattern(s): ${globList}\nWorking directory: ${process.cwd()}\n`,
```